### ISSUE TITLE
Clearly define the "news tag" tag? / category? / taxonomy? on Posts

### ISSUE THREAD

**Issue by hackajesse** [CityOfBoston/boston.gov#220]

**Comment by asalsman** [comment]
tl;dr: News Tags currently have no functional use. As such, "Tags" remains an acceptable term for them.

“News Tags” as a nomenclature is actually the most accurate title rather than if it were renamed. Currently, News Tags allow you to tag content with multiple tags. These tags display at the bottom of content as plain text, not linked.

Tag pages do not seem to exist, one cannot search by tags in the admin interface, and tags do nothing in terms of taxonomy. Tags actually seem redundant based on the Related Content fields which are available and actually can be used to sort and display content on other pages.

News Tags exist exclusively as part of Post meta. As such, even if we did have tag pages, only posts would show up; no events, public notices, etc.

On the flip side, if we had tag pages, we'd have an architectural nightmare because what's the difference between pages using news tags to say "Dorchester" as opposed to using related content's Neighborhood? In theory, a whole lot.

**Comment by hackajesse** [comment]
@matthewcrist ^^ something to think about for our search discussion

**Comment by hackajesse** [comment]
Thanks for clarifying this, @asalsman - closing this issue.
